mesa_frames/concrete/agentset.py

```
@@ -82,19 +82,76 @@ class AgentSet(AbstractAgentSet, PolarsMixin):
     _copy_only_reference: list[str] = ["_model", "_mask"]
     _mask: pl.Expr | pl.Series
 
-    def __init__(self, model: mesa_frames.concrete.model.Model) -> None:
+    def __init__(
+        self, model: mesa_frames.concrete.model.Model, name: str | None = None
```

Ruff (0.12.2)

Undefined name `mesa_frames`

(F821)